RecoLocalCalo/HcalRecProducers/src/ZdcSimpleReconstructor.cc

```diff
@@ -20,7 +20,9 @@ ZdcSimpleReconstructor::ZdcSimpleReconstructor(edm::ParameterSet const& conf):
   det_(DetId::Hcal),
   dropZSmarkedPassed_(conf.getParameter<bool>("dropZSmarkedPassed"))
 {
-  tok_input_ = consumes<ZDCDigiCollection>(conf.getParameter<edm::InputTag>("digiLabel"));
+  tok_input_castor = consumes<ZDCDigiCollection>(conf.getParameter<edm::InputTag>("digiLabelcastor"));
+  tok_input_hcal = consumes<ZDCDigiCollection>(conf.getParameter<edm::InputTag>("digiLabelhcal"));
```

Comment:
hi @slava77, I'm about to make a PR to 81X to fix this bug. Should I include edits to the other two files (EventFilter/CastorRawToDigi/interface/ZdcUnpacker.h and RecoLocalCalo/HcalRecProducers/src/ZdcSimpleReconstructor.h) as well, to make them match this 80X PR? That way only one PR for 81X and no more changes to this one.